unit_test/test_main.py

```diff
@@ -356,6 +359,44 @@ def test__new_job_submitted_handler__already_running_new_job(self) -> None:
         postgresql_if.put_new_job.assert_not_called()
 
 
+class PostgresJobManagerTest(unittest.TestCase):
+    def test__job_row_is_stale__returns_true(self) -> None:
+        # Arrange
+        orchestrator_config = OrchestratorConfig()
+        # Overwrite config setting
+        orchestrator_config.postgres_job_manager_config.job_retention_sec = 60
+        mocked_orchestrator = OrchestratorTest.MockedOrchestrator()
+        postgresql_if = mocked_orchestrator.postgresql_if
+        postgres_job_manager = PostgresJobManager(postgresql_if,
+                                                  orchestrator_config.postgres_job_manager_config)
+        cur_time = datetime.now()
+        job = JobDB()
+        job.registered_at = cur_time - timedelta(seconds=65)
+
+        # Act
+
+        # Assert
+        self.assertTrue(postgres_job_manager.job_row_is_stale(job=job, ref_time=cur_time))
+
+    def test__job_row_is_stale__returns_false(self) -> None:
+        # Arrange
+        orchestrator_config = OrchestratorConfig()
+        # Overwrite config setting
+        orchestrator_config.postgres_job_manager_config.job_retention_sec = 60
+        mocked_orchestrator = OrchestratorTest.MockedOrchestrator()
+        postgresql_if = mocked_orchestrator.postgresql_if
+        postgres_job_manager = PostgresJobManager(postgresql_if,
+                                                  orchestrator_config.postgres_job_manager_config)
+        cur_time = datetime.now()
+        job = JobDB()
+        job.registered_at = cur_time - timedelta(seconds=55)
+
+        # Act
+
+        # Assert
+        self.assertFalse(postgres_job_manager.job_row_is_stale(job=job, ref_time=cur_time))
+
+
```

Comment:
Please add the test cases to also cover `submitted_at` and `running_at` available on JobDB

Reply:
Extended in the new commit.